packages/blockchain/src/db/manager.ts

```diff
@@ -75,7 +75,7 @@ export class DBManager {
       const signerStates = await this.get(DBTarget.CliqueSignerStates)
       const states = (<any>rlp.decode(signerStates)) as [Buffer, Buffer[]]
       return states.map((state) => {
-        const blockNum = new BN(state[0])
+        const blockNum = toType(state[0], TypeOutput.BigInt)
```

Comment:
when we know this is a Buffer, instead of using toType, use bufferToBigInt. also for the other two cases below (L98, L120)